pipeline/test_extract.py

```diff
@@ -4,10 +4,9 @@
 
 import pytest
 import os
-from unittest.mock import patch
-from extract import get_api_request, fetch_api_data, export_api_data_to_csv, run_extract, get_time_offset
-
-pytest.skip(allow_module_level=True)
+import json
+from unittest.mock import patch, mock_open
+from extract import get_api_request, fetch_api_data, api_data_to_rows_and_columns, save_to_csv, run_extract, get_time_offset
```

Comment:
Small change here for readability:
```suggestion
from extract import (
    get_api_request, 
    fetch_api_data, 
    api_data_to_rows_and_columns, 
    save_to_csv, run_extract, 
    get_time_offset
)
```